lib/Elastica/Tool/CrossIndex.php

```diff
@@ -65,7 +65,7 @@ public static function reindex(
         array $options = array()
     ) {
         // prepare search
-        $search = new Search($oldIndex->getClient());
+        $search = $oldIndex->createSearch();
```

Comment:
Unrelated note, but I think this functionality will become obsolete with the new Reindex API coming in ES 2.3